src/Propel/Common/Util/PathTrait.php

```diff
@@ -36,7 +36,15 @@ protected function getTemplatePath(string $path): string
     {
         $srcPos = strpos($path, DIRECTORY_SEPARATOR . 'src' . DIRECTORY_SEPARATOR);
         if ($srcPos === false) {
```

Comment:
I am probably misunderstanding this, but above, it says templates can be on the same level as src folder, i.e. `./Propel2/templates`, which does not contain `/src/`. Wouldn't that throw an error here, even though it is a valid path?